Package.swift

```diff
@@ -26,8 +26,8 @@ let package = Package(
         .library(name: "SpeziLLMFog", targets: ["SpeziLLMFog"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ml-explore/mlx-swift", .upToNextMinor(from: "0.21.2")),
-        .package(url: "https://github.com/ml-explore/mlx-swift-examples", exact: "2.21.2"),  // Pin MLX Swift Examples as it doesn't follow semantic versioning
+        .package(url: "https://github.com/ml-explore/mlx-swift", .upToNextMinor(from: "0.25.5")),
+        .package(url: "https://github.com/ml-explore/mlx-swift-examples", exact: "2.25.5"),  // Pin MLX Swift Examples as it doesn't follow semantic versioning
```

Comment:
Thanks for lifting this, great to see that there are no breaking changes in their API